### PR TITLE
Refactor path for configuration file, opton(s) and Docker image

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,6 +25,7 @@ RUN make all
 
 FROM alpine:3.20 AS amf
 
+LABEL maintainer="Aether SD-Core <dev@lists.aetherproject.org>"
```
Comment (thakurajayL) [inline Dockerfile:28]: Could you pls send test mail on this.

File: factory/factory.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func InitConfigFactory(f string) error {
 	return nil
 }
 
-func UpdateAmfConfig(f string) error {
+func UpdateConfig(f string) error {
```
Comment (thakurajayL) [inline factory/factory.go:46]: Though I agree with this change there are plenty of variables with name amfConfig... But I do not want those many variable changes.

Comment (gab-arrobo) [reply factory/factory.go:46]: We can make those changes in another PR such that a single does not become to big to review. What do you think?

File: Dockerfile (hunk 1 above)
Comment (gab-arrobo) [reply Dockerfile:28]: Done. I just sent an email there. One caveat is that you need to subscribe to send/receive emails

Comment (thakurajayL) [reply Dockerfile:28]: Is that information captured somewhere? How to subscriber for mailer?

File: factory/factory.go (hunk 2 above)
Comment (thakurajayL) [reply factory/factory.go:46]: Changing function name is enough. Sometimes its easy to keep amf in the variable name. I would not recommend that change.

**Review by thakurajayL — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (gab-arrobo) [reply Dockerfile:28]: I do not see that being documented anywhere. As a matter of fact, I just signed up to the mailing list

**Pull request merged**